### ISSUE TITLE
Postgresql jar not loaded by commons-dbcp

### ISSUE THREAD

**Issue by alerat** [dependency-check/DependencyCheck#3412]
**Describe the bug**
We are using an external nist database and we put the postgresql driver in a directory.

The maven settings we use :

  ```
<profiles>
    <profile>
      <id>dependency-check</id>
      <activation>
        <activeByDefault>false</activeByDefault>
      </activation>
      <properties>
        <autoUpdate>false</autoUpdate>
        <centralAnalyzerEnabled>false</centralAnalyzerEnabled>
        <nexusAnalyzerEnabled>false</nexusAnalyzerEnabled>
        <ossindexAnalyzerEnabled>false</ossindexAnalyzerEnabled>
        <retireJsAnalyzerEnabled>false</retireJsAnalyzerEnabled>
        <databaseDriverPath>/usr/local/lib</databaseDriverPath>
        <databaseDriverName>org.postgresql.Driver</databaseDriverName>
        <connectionString>XXXXX</connectionString>
        <serverId>nist-db</serverId>
        <format>ALL</format>
      </properties>
    </profile>
</profiles>
```

The driver is not present in the classpath and Commons DBCP cannot use it.  It's ok in 6.1.6

**Version of dependency-check used**
maven plugin 6.2.0

**Log file**
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  01:23 min
[INFO] Finished at: 2021-06-01T12:24:00+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.owasp:dependency-check-maven:6.2.0:check (default-cli) on project xxxx: Fatal exception(s) analyzing xxxx: One or more exceptions occurred during analysis:
[ERROR] 	DatabaseException: Unable to connect to the dependency-check database
[ERROR] 		caused by DatabaseException: Error connecting to the database
[ERROR] 		caused by SQLException: Cannot load JDBC driver class 'org.postgresql.Driver'
[ERROR] 		caused by ClassNotFoundException: org.postgresql.Driver

**To Reproduce**
Try loading the postgresql driver from directory and not classpath

**Expected behavior**
Run analysis

**Additional context**
Commons DBCP try to load the postgresql driver from classpath and not from directory

**Comment by jeremylong** [comment]
Can you try configuring the plugin with a dependency like this:

```
        <plugins>
            <plugin>
                <groupId>org.owasp</groupId>
                <artifactId>dependency-check-maven</artifactId>
                <version>6.2.0</version>
                <dependencies>
                    <dependency>
                        <groupId>org.postgresql</groupId>
                        <artifactId>postgresql</artifactId>
                        <version>42.2.20</version>
                    </dependency>
                </dependencies>
```

**Comment by alerat** [comment]
It works but we have to modify our pom.xml files for all of our projects. We centralize all the dependency-check maven plugin configuration in a common settings.xml in our continuous integration.

Some of the code uses Commons DBCP and some not. Only the part with commons dbcp doesn't work

**Comment by jeremylong** [comment]
This should be fixed with the pending 6.2.1 release.

**Comment by alerat** [comment]
It works Thanks
